tests/integration/web/maintenance/views_test.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (C) 2022 Sikt
+#
+# This file is part of Network Administration Visualized (NAV).
+#
+# NAV is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License version 3 as published by
+# the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.  You should have received a copy of the GNU General Public
+# License along with NAV. If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+class TestMaintenanceCalendarView:
+    def test_calendar_renders_when_no_arguments_given(self, client):
+        response = client.get('/maintenance/', follow=True)
```

Comment:
```suggestion
        response = client.get('/maintenance/calendar/', follow=True)
```

Reply:
The calendar *always* generates a view for the current month when no arguments are given.

Whether you visit `/maintenance/` or `/maintenance/calendar/` is moot. Visiting `/maintenace/` without arguments, automatically redirects the client to `/maintenance/calendar/`, which is why the test includes the `follow=True` argument.